### PR TITLE
added responsive NavBar( not completed yet some styling issues )

### LINKED ISSUE #29: Add NavBar component to Sudoswap-frontend
### The Navbar consists of the following elements:

- Home page
- CreatePool page
- Swap transaction page

### PR DESCRIPTION
**Pull request by bisbist**
Completed: Added simple Responsive NavBar

TODO: There are some styling issues that I have not fixed yet, I will complete those after i complete my other tasks.

### CODE REVIEW

**Review by PantaAastha — COMMENTED** [review]
Why are we creating components inside utils folder? Wouldn't it be clearer if we create component inside components folder, something like components > navbar?

File: webui/src/App.js [hunk 1]
```diff
@@ -18,8 +22,11 @@ function App() {
 
   return initialized ? (
     <BrowserRouter>
+    <NavBar />
       <Routes>
         <Route path="/" element={<Home />} />
+        <Route path="/Pools" element={<PoolCreator />} />
+        <Route path="/Swap" element={<Swap />} />
```
Comment (PantaAastha) [inline webui/src/App.js:29]: I think `path=/pools` and `path=/swap` will be better.

Comment (bisbist) [reply webui/src/App.js:29]: Ok, I will correct that. Thanks for the review.

File: webui/src/Utils/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
+
```
Comment (PantaAastha) [inline webui/src/Utils/Navbar.js:1]: Please remove this

**Comment by bisbist** [comment]
> Why are we creating components inside utils folder? Wouldn't it be clearer if we create a component inside the components folder, something like components > navbar?

Oh, yes we can do that. That is a blunder mistake.

File: webui/src/App.js [hunk 3]
```diff
@@ -18,8 +22,11 @@ function App() {
 
   return initialized ? (
     <BrowserRouter>
+    <NavBar />
       <Routes>
         <Route path="/" element={<Home />} />
```
Comment (PantaAastha) [inline webui/src/App.js:27]: Can you try something like this?
`<Route path="/" element={<Navigate to="/pools" replace />} />` This should redirect the user to pools page.

File: webui/src/App.js [hunk 4]
```diff
@@ -3,6 +3,10 @@ import { BrowserRouter, Routes, Route } from "react-router-dom";
 import { setConfig } from "./config";
 import Home from "./pages/Home";
 import "@blueprintjs/core/lib/css/blueprint.css";
+import PoolCreator from "./components/PoolCreator";
+import NavBar from "./Utils/Navbar";
+import Swap from "./components/Swap";
+
```
Comment (PantaAastha) [inline webui/src/App.js:9]: import { Navigate } from "react-router-dom"

File: webui/src/App.js (hunk 3 above)
Comment (bisbist) [reply webui/src/App.js:27]: Ok, I will try that out.

**Review by PantaAastha — APPROVED** [review]

**Pull request merged**